src/main.js

```diff
@@ -224,14 +328,15 @@ export async function insertBadges ({ forceReload } = {}) {
   const pages = Math.ceil(dois.length / BATCH_SIZE)
   for (let i = 0; i < pages; i++) {
     const currentDOIs = dois.slice(i * BATCH_SIZE, (i + 1) * BATCH_SIZE)
-    const [{ tallies }, { notices }] = await Promise.all([fetchTallies(currentDOIs), fetchNotices(currentDOIs)])
+
+    const [{ tallies }, { notices }, { tallies: sectionTallies }] = await Promise.all([fetchTallies(currentDOIs), fetchNotices(currentDOIs), fetchSectionTallies(currentDOIs)])
```

Comment:
can we construct this depending on the config?
That way folks who only want citation type tallies and not section tallies will not have to have the extra network calls and vice versa.

Reply:
oh didn't see `Maybe more efficient tally fetching so we only get section tally info if needed` lol

Reply:
Yep

Reply:
Okay @domenicrosati  here is what I did:

- Get the `badgesToLoad`
- For each badge, `getDOI`
- If the badge has `tallyShow`, push it into the list of DOIs to fetch tallies for
- If the badge has `sectionTallyShow`, push it into the list of DOIs to fetch section tallies for
- In the for loop where we fetch, set the `pageSize` to be the larger page size of the two lists
- Iterate in batch size, fetch the tallies, and load them
- If we exceed the smaller list's page size, `slice()` will return `[]`, and the tallies won't be fetched
- When inserting the badge, check to make sure the `doi` is in either the `tallies` or `sectionTallies` keys

I opted to do it this way so we still have one loop where it happens at the same time, and we don't have weird issues with the `fetched` attribute on the badge.